### PR TITLE
Client side

### LINKED ISSUE #15: Client update error receive & logical statements
Follow the protocol uniformisation

### PR DESCRIPTION
**Pull request by BetterBlood**
Implements protocol update, some logical statement to fit server answer

- #15

### CODE REVIEW

**Comment by BetterBlood** [comment]
link with pull request : #19

File: src/main/java/ch/heigvd/Client.java [hunk 1]
```diff
@@ -103,21 +104,31 @@ public void run() {
...
                     } else {
                         System.out.println("[Client] correct letter ! \n" + nbrLives + " lives\n" + answer[1]);
                     }
-
                     word = answer[1];
                 } else if (answer[0].equalsIgnoreCase(WRONG)) {
                     System.out.println("[Client] incorrect letter ! \n" + --nbrLives + " lives\n" + word);
                 } else if (answer[0].equalsIgnoreCase(WIN)) {
-                    System.out.println("[Client] Congratulations ! The correct word was " + answer[1]);
+                    System.out.println("[Client] Congratulations ♥_♥ ! The correct word was " + answer[1]);
```
Comment (GuggisbergSimon) [inline src/main/java/ch/heigvd/Client.java:129]: ♥

File: src/main/java/ch/heigvd/Client.java [hunk 2]
```diff
@@ -66,16 +67,16 @@ public void run() {
                             if (!isNumeric(arguments[1])) {
                                 command = defaultStart;
                             } else {
-                                command = START + " " + DEFAULT_LANGUAGE;
-                            }
+                                command = START + " " + DEFAULT_LANGUAGE; // TODO : ?????
```
Comment (GuggisbergSimon) [inline src/main/java/ch/heigvd/Client.java:70]: good catch, the intended was to have ```START + " " + arguments[1] + " " + DEFAULT_LANGUAGE``` probably

File: src/main/java/ch/heigvd/Client.java [hunk 3]
```diff
@@ -52,9 +52,10 @@ public void run() {
                     continue;
                 }
 
-                if (word.isEmpty() && arguments[0].equalsIgnoreCase(START)) {
+                if (arguments[0].equalsIgnoreCase(START)) { // TODO : pk IgnoreCase ? (toUpperCase ligne 47)
```
Comment (GuggisbergSimon) [inline src/main/java/ch/heigvd/Client.java:55]: probably a superfluous check that could be replaced by ```arguments[0].equals(START)``` I wasn't sure how we wanted to handle characters such as éèà etc

**Review by GuggisbergSimon — APPROVED** [review]
see the server PR comments as both go hand in hand

**Pull request merged**